test/langtools/tools/javac/processing/model/util/elements/TestGetDocComments.java

```diff
@@ -0,0 +1,185 @@
+        }
+    }
+
+    @interface ExpectedComment {
+        String value();
+    }
+
+    // Basic processing of interior lines
+    /**
+     *Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+     *eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+     *enim ad minim veniam, quis nostrud exercitation ullamco laboris
+     *nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
+     *in reprehenderit in voluptate velit esse cillum dolore eu
+     *fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+     *proident, sunt in culpa qui officia deserunt mollit anim id est
+     *laborum.
+     */
+    @ExpectedComment("""
+     Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+     eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+     enim ad minim veniam, quis nostrud exercitation ullamco laboris
+     nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
+     in reprehenderit in voluptate velit esse cillum dolore eu
+     fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+     proident, sunt in culpa qui officia deserunt mollit anim id est
+     laborum.
+      """)
+    // End-of-line-style comment
+    @SuppressWarnings("") // A second preceding annotation
+    private void foo() {return ;}
+
+
+    // Check removal of various *'s and space characters;
+    // use Unicode escape to test tab removal
+    /**
+*Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+**eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+***enim ad minim veniam, quis nostrud exercitation ullamco laboris
+*****nisi ut aliquip ex ea commodo consequat.
+ \u0009*Duis aute irure dolor in reprehenderit in voluptate velit esse
+ **cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+  ***cupidatat non proident, sunt in culpa qui officia deserunt mollit
+                                            *anim id est laborum.
+     */
+    @ExpectedComment("""
+       Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+       eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+       enim ad minim veniam, quis nostrud exercitation ullamco laboris
+       nisi ut aliquip ex ea commodo consequat.
+       Duis aute irure dolor in reprehenderit in voluptate velit esse
+       cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+       cupidatat non proident, sunt in culpa qui officia deserunt mollit
+       anim id est laborum.
+       """)
+    @SuppressWarnings("") // A second preceding annotation
+    // End-of-line-style comment
+    private void bar() {return ;}
+
+    // Spaces _after_ the space-asterisk prefix are _not_ deleted.
+    /**
+     * Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+     * eiusmod tempor incididunt ut labore et dolore magna aliqua.
+     */
+    @ExpectedComment( // Cannot used a text block here since leading spaces are removed
```

Comment:
You could use a text block, by carefully adjusting the indentation of the trailing `"""`